auto_gptq/modeling/_base.py

```diff
@@ -370,28 +368,10 @@ def tmp(_, inp, out):
                             static_groups=self.quantize_config.static_groups,
                         )
 
-                        stat = {"layer": i + 1, "module": name, "avg_loss": avg_loss, "time": duration}
+                        stat = {"layer": i, "module": name, "avg_loss": avg_loss, "time": duration}
```

Comment:
@fxmarty  Sorry if duplicate. I can't find the original msg for some reason.  Can we keep `i + 1` index notation? This stat line is printed at each layer/module and at the very end (summary) but the logs are interjected with tqdm progress logs as well and they using `i + 1` index.  This stat is meant for visual consumption and having it be consistent with tqdm make the logs less confusing to the eye.

Reply:
yes yes let's keep it - should have deleted my comment instead of closing